scheds/rust/scx_tickless/src/bpf/main.bpf.c

```diff
@@ -643,7 +664,13 @@ int enable_primary_cpu(struct cpu_arg *input)
 
 s32 BPF_STRUCT_OPS_SLEEPABLE(tickless_init)
 {
-	return scx_bpf_create_dsq(SHARED_DSQ, -1);
+	int ret;
+
+	ret = scx_bpf_create_dsq(SHARED_DSQ, -1);
+	if (ret < 0)
+		return ret;
+
+	init_timer(bpf_get_smp_processor_id());
```

Comment:
I guess the other way to init the timer would be use another prog timer and init from userspace. Then you could manually set the affinity and have a way of directly placing a timer on a particular CPU.